src/resources/i18n/translations/en.json

```
@@ -410,7 +409,7 @@
 	"No downloads in progress": "Nothing is being downloaded",
 	"No episodes found": "Could not find any episodes",
 	"No history items found": "Could not find any history",
-	"No internet connection": "Disconnected from the Internet",
+	"No internet connection": "No internet connection",
```

```suggestion
	"No internet connection": "No connection to the Internet",
```
This allows there to be a capital i in "Internet" in all languages. Getting that part wrong looks terrible.

---

👍